src/PKSim.UI.Starter/ApplicationStartup.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Globalization;
+using System.Threading;
+using OSPSuite.Assets;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Core.Journal;
+using OSPSuite.Core.Services;
+using OSPSuite.Infrastructure.Import.Services;
+using OSPSuite.Presentation;
+using OSPSuite.Presentation.Core;
+using OSPSuite.Presentation.DTO;
+using OSPSuite.Presentation.Services;
+using OSPSuite.Presentation.Views;
+using OSPSuite.UI.Mappers;
+using OSPSuite.UI.Services;
+using OSPSuite.Utility.Container;
+using OSPSuite.Utility.Exceptions;
+using PKSim.CLI.Core.MinimalImplementations;
+using PKSim.Core;
+using PKSim.Core.Services;
+using PKSim.Infrastructure;
+using PKSim.Presentation;
+using PKSim.Presentation.DTO.Mappers;
+using PKSim.Presentation.Presenters.ExpressionProfiles;
+using PKSim.Presentation.Services;
+using PKSim.Presentation.Views.ExpressionProfiles;
+using PKSim.UI.Views.ExpressionProfiles;
+
+namespace PKSim.UI.Starter
+{
+   public static class ApplicationStartup
+   {
+      static IContainer _container;
+
+      public static IContainer Initialize(IShell shell)
+      {
+         if (_container != null)
+            return _container;
+
+         _container = InitializeForStartup(shell);
+
+         return _container;
+      }
+
+      public static IContainer InitializeForStartup(IShell shell)
```

Comment:
This is cool - we are actually registering the MoBi main view as IShell and then all the sub-views are shown correctly because IoC makes sure they are using MoBi as the Parent view.